### PR TITLE
New Crontab Cheat Sheet -- convert from HTML to JSON

### LINKED ISSUE #946: Crontab Cheatsheet: Bottom text is getting cut-off
screen shot 2015-01-21 at 2 51 51 pm
https://duck.co/ia/view/crontab_cheat_sheet

### PR DESCRIPTION
**Pull request by Sayanc93**
IA :https://duck.co/ia/view/crontab_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Crontab Cheat Sheet](https://duck.co/ia/view/crontab_cheat_sheet)

**Description**: Crontab cheat sheet

**Example Query**: [crontab help](https://beta.duckduckgo.com/?q=crontab help), [crontab cheat sheet, crontab example](https://beta.duckduckgo.com/?q=crontab cheat sheet, crontab example)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/crontab_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/crontab_cheat_sheet)*

**Comment by Sayanc93** [comment]
can this be merged?

**Comment by moollaza** [comment]
Deploying to beta for review...

File: share/goodie/cheat_sheets/json/crontab.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "name": "Crontab",
+    "description": "displayed as subtitle of the AnswerBar; optional",
+
+    "metadata": {
+        "sourceName": "displayed at the bottom of the AnswerBar; optional",
+        "sourceUrl" : "https://url.to.the.source"
+    },
+
+    "aliases": [
+        "crontab cheat sheet, cron tab"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/crontab.json]: Remove the first alias.

File: share/goodie/cheat_sheets/json/crontab.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        }, {
+            "key" : "5 12-18/2 * * * /path/to/command",
+            "val" : "Run at 12:05, 13:05, ..., and 18:05"
+        }, {
+            "key" : "0 0 1 * * /path/to/command",
+            "val" : "Run on the first day of every month, at 12:00am"
+        }, {
+            "key" : "0 0 1 */3 * /path/to/command",
+            "val" : "Run on the first day of every third month, at 12:00am"
+        }]
```
Comment (jdorweiler) [inline share/goodie/cheat_sheets/json/crontab.json]: You're missing a `}` here

File: share/goodie/cheat_sheets/json/crontab.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
+{
+    "id": "crontab_cheat_sheet",
+    "name": "Crontab",
+    "description": "displayed as subtitle of the AnswerBar; optional",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/crontab.json]: We need to update this to reflect the cheat sheet.

File: share/goodie/cheat_sheets/json/crontab.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+    "metadata": {
+        "sourceName": "displayed at the bottom of the AnswerBar; optional",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/crontab.json]: It looks like you haven't updated the template text here (and elsewhere).

**Comment by jdorweiler** [comment]
This pull request is being closed due to inactivity but, if anyone would like to continue it, please @ mention 'duckduckhack-contributors' for help and http://docs.duckduckhack.com/submitting/submitting-overview.html for reference.

**Pull request closed without merging**